### ISSUE TITLE
limb amputation

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10962]
**Preferred term label:**
see issue #10960

**Synonyms**
upper limb amputation
lower limb amputation

**Definition (free text, please give PubMed ID)**
PMID: 25531236 Amniotic band syndrome (ABS) encompasses a wide spectrum of disease of uncertain etiology. The presentation of the disease is dependent on the anatomic location of the bands, where bands affecting the extremities can lead to amputation and pseudosyndactyly, while broad bands can result in fatal conditions such as the limb-body wall complex (LBWC)

PMID: 25531236 Extremities were clearly the most commonly affected site. The presentation for extremity involvement can be late, after amputation has already occurred, or early when there is still an opportunity for fetoscopic band release for limb salvage.

PMID: 31424867. There is no known clear etiology of ABS. However, it is thought that rupture of amnion early in pregnancy results in the development of multiple loose strands (amniotic bands) that adhere to or entangle the fetus. Constriction of otherwise normally developing organs results in constriction rings, some severe cases leading to vascular disruption and could potentially result in **amputation of the involved anatomic structure**. 
(...) Constriction rings and **limb or digital amputation** are the most common findings, present in at least 80 percent of cases. The amniotic bands may be present, confined to the skin or soft tissue, and extend deep into the tissue. If constriction results in **amputation in utero**, then the amputated part is usually resorbed and not visible following birth.

**Parent term (use hpo.jax.org/app)**
Congenital amputation see issue #10960

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM 217100 CONSTRICTING BANDS, CONGENITAL
ORPHA:295000 Amniotic band syndrome

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 

Adding new term: 
- ID: HP:6001391
- label: Intrauterine limb amputation
- parents: [Abnormal fetal skeletal morphology (HP:0025662)](https://hpo.jax.org/app/browse/term/HP:0025662)
- pmid: PMID:31424867|PMID:31407870|PMID:31787161
- def: A developmental anomaly characterized by partial or complete absence of an arm or leg at birth, resulting from interruption of normal development in utero.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10962
- synonyms:
